### PR TITLE
[azure-multiadpi-storagev2] Add support for storage data plane track2 SDK

### PR DESCRIPTION
**Pull request by Juliehzl**
Add support for storage data plane track2 SDK with new package azure-multiapi-storagev2:
- blob
    - v2019-02-02
    - v2019-07-07
- filedatalake
    - v2018-11-09
- fileshare
    - v2019-02-02
    - v2019-07-07
- queue
    - v2018-03-28

### CODE REVIEW

File: scripts/updatev2_1.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    tgt=../azure/multiapi/storagev2/$service/v$ver
+    mkdir -p $tgt
+    src=$src_root/$service
+    cp -R $src $tgt
+
+    for f in `find $tgt/$service -name '*.py'`; do
+        echo Updating $f
+        # remove BOM
+        sed -i '1s/^\xEF\xBB\xBF//' $f
```
Comment (haroldrandom) [inline scripts/updatev2_1.sh:33]: Are you sure about it has BOM?

Comment (Juliehzl) [reply scripts/updatev2_1.sh:33]: not sure. we have such check in track1 SDK.

File: README.rst [hunk 2]
```diff
@@ -17,6 +17,17 @@ Handles multi-API versions of Azure Storage Data Plane originally from https://g
 
 Change Log
 ----------
+0.3.0
++++++
+* Add support for storage data plane track2 SDK with new package azure-multiapi-storagev2:
+    - blob
+	- v2019-07-07
```
Comment (qianwens) [inline README.rst:24]: >- v2019-07-07 [](start = 1, length = 13)

add indent before - v2019-07-07 ?

File: README.rst [hunk 3]
```diff
@@ -17,6 +17,17 @@ Handles multi-API versions of Azure Storage Data Plane originally from https://g
...
+    - filedatalake
+        - v2018-11-09
+    - fileshare
+        - v2019-07-07
+    - queue
+	- v2018-03-28
```
Comment (qianwens) [inline README.rst:30]: >v2018-03-28 [](start = 3, length = 11)

same as above

**Comment by haroldrandom** [comment]
ADO didn't show, could you rebase it or operate setting on Github settings to make it auto merge with latest master and then show ADO checks?

File: README.rst (hunk 2 above)
Comment (Juliehzl) [reply README.rst:24]: Sure.

**Pull request merged**